lms/static/js/commerce/views/receipt_view.js

```diff
@@ -61,6 +63,18 @@ var edx = edx || {};
                 $courseNamePlaceholder.text(responseData.name);
             });
         },
+        renderUserFullNamePlaceholder: function (username) {
+            var userModel = Backbone.Model.extend({
```

Comment:
Yeah, because this page isn't using require the way you have implemented it here is the best way. Once the page has been refactored you can move this to an external file and use RequireJS to access it.